libs/application/templates/driving-instructor-registrations/src/fields/StudentsOverview/pagination.ts

```diff
@@ -0,0 +1,15 @@
+export const PAGE_SIZE = 10
+
+export function paginate(students: any, pageSize: number, pageNumber: number) {
```

Comment:
Would be better if we could type students. You can use union types to define object or undefined.

Reply:
Added to backlog :)